README.md

```diff
@@ -83,17 +83,23 @@ root@2efa5b50b909:
 
 ### Examples and Tutorials
 
-The following use cases can be found in our [API documentation examples section](https://nvidia.github.io/NVTabular/main/examples/index.html):
-* Preprocessing
-* Feature engineering
-* Dataloading in Tensorflow, PyTorch, and HugeCTR
-
-Performance of the Criteo DRLM workflow demonstrates the effectiveness of the NVTabular library. The original ETL script provided in Numpy took over five days to complete. Combined with CPU training, the total iteration time is over one week. By optimizing the ETL code in spark and running on a DGX-1 equivalent cluster, we were able to bring that time down to three hours for ETL and one hour for training.
```

Comment:
I like these two paragraphs about accelerating criteo - can we leave them in? I agree that we can remove the rossmann one though